music/src/core/compat/global.ts

```diff
@@ -31,3 +37,18 @@ export interface Performance {
 export const fetch: typeof window.fetch = isNode ? require('node-fetch') : window.fetch.bind(window);
 export const performance: Performance = isNode ? require('./performance_node') : window.performance;
 export const navigator = isNode ? require('./navigator_node') : window.navigator;
+
+export function isSafari() {
+  if (isNode) {
+    throw new Error('Cannot check if Safari in Node.js');
+  }
+  // tslint:disable-next-line:no-any
+  return (window as any).safari !== undefined;
```

Comment:
whoaaaa where is this getting set? i think if we're setting this it's not super great, because it's a very vague noun on the window object so there is a chance in the future someone can accidentally break this

Reply:
No, this is another sweet thing that I didn't know existed until I did some research. Seems like a simpler way to check than parsing `navigator.userAgent`

Reply:
Hmmmmm this isn’t in the spec (https://developer.mozilla.org/en-US/docs/Web/API/Window). Will it be true on chrome on iOS, or other webview on iOS? I really don’t know who is setting this and when, and I don’t trust it, I’m sorry. Can we revert to the tried and true navigator.userAgent please?